bootstrap.d/media-libs.yml

```diff
@@ -799,8 +799,8 @@ packages:
     source:
       subdir: 'ports'
       git: 'https://gitlab.freedesktop.org/mesa/mesa.git'
-      tag: 'mesa-23.0.2'
-      version: '23.0.2'
+      tag: 'mesa-23.1.9'
+      version: '23.1.9'
       tools_required:
```

Comment:
This seems erroneous. Can you remove this?

Reply:
Was indeed erroneous, removed